doc/locale/ja/LC_MESSAGES/contribution/documentation/i18n.po

```diff
@@ -56,20 +56,23 @@ msgstr "Sphinxをインストールします。（インストールされてい
 msgid "Clone Groonga repository."
 msgstr "Groongaのリポジトリをcloneします。"
 
+msgid "Update .edit files"
+msgstr ".editファイルを更新します。"
+
+msgid "Edit .edit files"
+msgstr ".editファイルを編集します。"
+
 msgid "Update .po files."
 msgstr ".poファイルを更新します。"
 
-msgid "Edit .po files."
-msgstr ".poファイルを編集します。"
-
 msgid "Generate HTML files."
 msgstr "HTMLファイルを生成します。"
 
 msgid "Confirm HTML output."
 msgstr "HTMLの出力を確認します。"
 
-msgid "Repeat 2.-4. until you get good result."
-msgstr "翻訳が完了するまで、2.-4.を繰り返します。"
+msgid "Repeat 3.-7. until you get good result."
+msgstr "翻訳が完了するまで、3.-7.を繰り返します。"
```

Comment:
Because the existing translation of this document translated "翻訳" into "good result".
(I thought it is not a good translation, though...)

Okay, I will change them like below.

`I will until you finish to translate.`

Reply:
I think it is good fix! 
Because Groonga is a search tool, a word "good result" make me think of the search result.